### DISCUSSION TITLE
Cannot find neither AddBatchStatus nor AddStatusService

### DISCUSSION THREAD

**Discussion by AxReds** [yaleman/pvoutput#285] [General]
Hi @yaleman,

I'm trying to add data to my PVOutput.org account using your library but I cannot find a way to add anything as it seems I'm not able to invoke the **_addbatchstatus_** method nor an **_addstatusservice_** function exists even if they are both present as web api from PVOutput.org.

For **_addbatchstatus_** the error I get is:
```Exception has occurred: AttributeError
'PVOutput' object has no attribute 'addbatchstatus'
  File "PVOut Uploader.py", line 31, in <module>
    response = pvo.addbatchstatus(data=data)
               ^^^^^^^^^^^^^^^^^^
AttributeError: 'PVOutput' object has no attribute 'addbatchstatus'
```

What am I missing?

Thanks Ax

**Comment by yaleman** [comment]
`addbatchstatus` is not implemented because I've never needed it. `addstatus` is available and works on my systems though.

I'm pushing a test branch `yaleman/issue285` - if you want to try the code from there and offer feedback that'd be much appreciated!
